airbyte-webapp/src/components/StatusIcon/CircleLoader.tsx

```diff
@@ -38,8 +38,8 @@ const CircleLoader = ({ title }: Props): JSX.Element => (
         gradientUnits="userSpaceOnUse"
         gradientTransform="translate(0 0)"
       >
-        <stop id="eDwmAshgIQE3-fill-0" offset="17.9167%" stop-color="#d1d1db" />
-        <stop id="eDwmAshgIQE3-fill-1" offset="100%" stop-color="rgba(209,209,219,0)" />
+        <stop id="eDwmAshgIQE3-fill-0" offset="17.9167%" stopColor="#d1d1db" />
```

Comment:
the console told me we should be using the camelCase version of this

Reply:
Fixed in #12513